influxdb-watcher/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# librdkafka is not available in a stable alpine release
```

Comment:
You don't need librdkafka so you can use a stable alpine

Reply:
Done